src/session.ts

```diff
@@ -265,6 +265,7 @@ export function trackBarfSessionStatistics(): void {
   // Also create a backup tracker for items
 
   if (marginalSession) marginalFamiliarsExcessTotal += marginalFamiliarsExcessValue;
+  marginalFamiliarsExcessValue = 0;
```

Comment:
This happens after the `if` statement. Checking if the intention is to reset back to `0` here.

Reply:
Yes. The code only sets `marginalFamiliarsExcessValue` when the `barfFamiliar()` code is called. If we do not reset it after adding this value to the total, then when we run a non-barf turn we would potentially continually add the excess value to the total, overaccounting for our marginal familiars (I guess technically we only call `trackBarfSessionStatistics()` when running barf so this shouldn't happen in theory, but I did hit a bug with overaccounting the other day so I'm in the process of figuring out what went wrong).

There's no harm always resetting it to 0 regardless of whether we are currently tracking the marginal session.

Reply:
Okay cool, that makes sense.

Reply:
I had a similar idea, but unfortunately that would run into some issues with circular imports unless we do a whole bunch of refactoring.

`session.ts` provides not only `garboValue()` and a few other functionalities which the other files, so I cannot have it import from `barfFamiliars.ts`; the values of the marginal familiars are computed within the `/familiars` code itself (which does use `garboValue` to evaluate the drops), so it's hard to extract the logic from within.

I agree it's not the cleanest solution for now, but I believe it would take a pretty big overhaul to clean stuff up, which is probably out of scope for now (I think).